### DISCUSSION TITLE
Hello Everyone!

### DISCUSSION THREAD

**Discussion by UnaritaemModey** [Cyfrin/foundry-full-course-cu#1656] [General]
Hey there. This is me beginning my adventures with Cyfrin updraft(Foundry101). I'm super grateful for the positivity displayed by this community so far. I'M really intentional about learning and interacting. Hope to connect with everyone!

**Comment by EngrPips** [comment]
Hello @UnaritaemModey, Welcome aboard, and I wish you the best on your journey.

**Reply by UnaritaemModey** [reply]
Thank you!

**Reply by EngrPips** [reply]
You are welcome, Friend.
